payment_signal/README.rst

```diff
@@ -0,0 +1,47 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl
+   :alt: License: AGPL-3
+
+==============
+Payment Signal
+==============
+
+Module to signal quotations and orders
+
+
+Usage
+=====
+
+This module creates a field to enter the payment signal,
+which is automatically calculated by a percentage associated with the company.
+These amounts, automatically appear in Invoice Order.
+
+.. image:: static/payment-signal-01.png
+
+.. image:: static/payment-signal-02.png
+
+.. image:: static/payment-signal-03.png
+
```

Comment:
Please add runbot link

Reply:
Sorry, how can I create the route of the rumbot link? Thank you.

Reply:
You can avoid managing those things creating readme directory and let readme bot do the job.
Please read https://github.com/OCA/maintainer-tools/pull/339 details